manifests/d/Dichromate/Browser/107.0.5304.122/Dichromate.Browser.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Dichromate.Browser
+PackageVersion: 107.0.5304.122
+PackageLocale: en-US
+Publisher: The Dichromate Authors
+PublisherUrl: https://github.com/NuclearDevelopers/Dichromate
+Author: Douloureux
+PackageName: Dichromate
+PackageUrl: https://github.com/NuclearDevelopers/Dichromate
+License: MIT License

```

```suggestion
License: BSD 3-Clause Clear License
```
Isn't the license BSD 3-Clause according to https://github.com/NuclearDevelopers/Dichromate/blob/main/LICENSE?